### PR TITLE
NAS-120382 / 22.12.3 / Set Protective MBR boot flag for legacy boot

### PR DESCRIPTION
**Pull request by usaleem-ix**
There have been some reports from users trying to install SCALE on HP Microserver Gen 8. Some platforms including HP Microserver, that have legacy bios, do not boot if this flag is not set.

This flag is present on the MBR on GPT table. EFI specification dictates this flag should not be set. sgdisk utility does not allow to set this flag. EFI system does not boot if this flag is set.

If legacy boot is detected, ask the user if they want to be able to boot in the EFI mode vs more compatible legacy. Set the flag if they select no.

### CODE REVIEW

**Review by yocalebo — CHANGES_REQUESTED** [review]
Hold off on merge for now, pretty sure this will break automation since we have expect scripts that parse console strings.

**Comment by usaleem-ix** [comment]
Sure.

**Review by yocalebo — APPROVED** [review]
QE team will change the expect script after this is merged.

**Pull request merged**
